src/commands/context/contextCommand.ts

```diff
@@ -0,0 +1,329 @@
+import SASjs from '@sasjs/adapter/node'
+import { readFile, ServerType } from '@sasjs/utils'
+import { CommandExample, ReturnCode } from '../../types/command'
+import { TargetCommand } from '../../types/command/targetCommand'
+import { getAuthConfig } from '../../utils'
+import { create } from './create'
+import { deleteContext } from './delete'
+import { edit } from './edit'
+import { exportContext } from './export'
+import { parseConfig } from './internal/parseConfig'
+import { validateConfigPath } from './internal/validateConfigPath'
+import { list } from './list'
+
+enum ContextSubCommand {
+  Create = 'create',
+  Delete = 'delete',
+  Edit = 'edit',
+  Export = 'export',
+  List = 'list'
+}
+
+const syntax = {
+  [ContextSubCommand.Create]: 'context <subCommand>',
+  [ContextSubCommand.Delete]: 'context <subCommand> <contextName>',
+  [ContextSubCommand.Edit]: 'context <subCommand> [contextName]',
+  [ContextSubCommand.Export]: 'context <subCommand> <contextName>',
+  [ContextSubCommand.List]: 'context <subCommand>'
+}
+const usage = {
```

Comment:
Documentation is now side-by-side with the actual code. Much easier to find and keep updated.